src/Amido.Stacks.Messaging.Azure.ServiceBus.Tests/UnitTests/Extensions/MessageExtensionsTests.cs

```diff
@@ -27,6 +30,22 @@ public void MessageExtensionSetEnclosedMessageType()
             message.UserProperties.Count.ShouldBe(1);
             message.UserProperties["EnclosedMessageType"].ShouldBe("Amido.Stacks.Messaging.Commands.NotifyCommand, Amido.Stacks.Messaging.Commands");
         }
+        
+        [Fact]
+        public void MessageExtensionSetSessionId()
+        {
+            var message = new Message();
+            message.SetSessionId(new NotifyEvent(Guid.NewGuid(), 1, "session-id"));
+            message.SessionId.Should().Be("session-id");
```

Comment:
I think this should be something like:
`message.SessionId.ShouldBe("session-id");`